geospaas/base_viewer/views.py

```diff
@@ -1,73 +1,51 @@
 from django.shortcuts import render
 from django.views.generic import View
 
-from geospaas.base_viewer.forms import TimeAndSourceForm, SpatialSearchForm
+from geospaas.base_viewer.forms import OverallForm
 from geospaas.catalog.models import Dataset as CatalogDataset
```

Comment:
Since you don;t define a new Dataset in this module, you don't have to import it with another name
```suggestion
from geospaas.catalog.models import Dataset
```